File: Calculadora/calculadora.py
print("Aló! bienvenidos a la caluladora")
print("Aquí las opciones: ")
print("1) Sumar")
print("2) Restar")
print("3) Multiplicar")
print("4) Dividir")
print("5) Salir")

ciclo=0;
print()

while ciclo == 0: 
    opcion = float(input("Qué gustas hacer? "))
    match opcion:
        case 1:
            num1=float(input("tu primer numero? "))
            num2=float(input("tu segundo numero? "))
            print("Resultado: "+str(num1+num2))
        case 2:
            num1=float(input("tu primer numero? "))
            num2=float(input("tu segundo numero? "))
            print("Resultado: "+str(num1-num2))
        case 3:
            num1=float(input("tu primer numero? "))
            num2=float(input("tu segundo numero? "))
            print("Resultado: "+str(num1*num2))
        case 4:
            num1=float(input("tu primer numero? "))
            num2=float(input("tu segundo numero? "))
            if num2>0:
                print("Resultado: "+str(num1/num2))
            else:
                print("No se puede dividir entre cero D:")
        case 5:
            ciclo=1;
            
        case _:
            print("Esa no es opción -_- ")

File: Calculadora/Calculadora.c
#include<stdio.h>

float numeros[2];
void menu(){
    printf("1. Suma\n");
    printf("2. Resta\n");
    printf("3. Multiplicación\n");
    printf("4. División\n");
    printf("5. Salir\n");
}

void leerData(){
    printf("\nPrimer numero: ");
    scanf("%f",&numeros[0]);
    printf("\nSegundo numero: ");
    scanf("%f",&numeros[1]);
}


int main(int argc, char const *argv[])
{
    printf("Aló! Bienvenidos a la calculadora simple :)\n");
    printf("Aquí las opciones: \n");
    int operacion,cic=1;
    while (cic==1){
        printf("\n**********************************************************\n");
        menu();
        scanf("%i",&operacion);
        switch (operacion)
        {
        case 1:
            leerData();
            printf("La suma es: %0.3f",(numeros[0]+numeros[1]));
        break;
        
        case 2:
            leerData();
            printf("La resta es: %0.3f",(numeros[0]-numeros[1]));
        break;
        
        case 3:
            leerData();
            printf("La multiplicación es: %0.3f",(numeros[0]*numeros[1]));
        break;

        case 4:
            leerData();
            if(numeros[1]!=0){
                printf("La división es: %0.3f",(numeros[0]/numeros[1]));
            }else{
                printf("No se puede dividir entre 0 :(");
            }
        break;

        case 5:
            printf("Adiós!");
            cic=0;
        break;

        default:
            printf("Opción no válida");
        }
    }
    
    return 0;
}

